include/PR/gbi.h

```diff
@@ -2575,6 +2575,8 @@ _DW({									\
  * uc_start  = ucode text section start
  * uc_dstart = ucode data section start
  */
+#define SP_UCODE_DATA_SIZE      0x800
```

Comment:
in oot, this constant is defined in PR/ucode.h which seems to be missing from this repo for w/e reason

Reply:
Yes, please bring in OoT's ucode.h instead of putting this here. Or if you aren't confident to set it up properly, put the define in the file with a `//! TODO` and I'll fix it later.